data/quick-builds/rebel-alliance/rebel-alliance.json

```diff
@@ -1,54 +1,50 @@
 {
   "quick-builds": [
     {
-      "xws": "qbbladesquadronveteran",'
```

Comment:
I think an XWS entery in each quick build is required for 2 major reasons.
1) It allows squad builders to more easily access specific data
2) It also allows for specification for ships with multilple quick build cards like the Black Squadron Tie/ln which has at least 2 different qb's. The correct method for defining xws can be found in the spec and the product sku is also in the QB PDF's.

Reply:
I've discussed this already with Guido and Eli (the spec author) and both of them have said that quick builds don't need a spec ID and the spec doesn't need to know about the quick build format. A quick build card is just a list of pilots and their upgrades, which is the same as an ordinary squad (which xws already defines). In Eli's words:

"so i agree that putting the quickbuild stuff into xwing-data2 makes sense, just that XWS doesn't need to know that it's a QB squad vs. 2nd ed squad vs. extended format squad vs. HotAC etc. 

pilots+upgrades is the end result, and XWS is source-neutral, except that the optional `points` field is a nod towards standard tournament building rules"
Regarding your points: quick build cards don't need a unique ID to be uniquely identifiable, because each quick build card contains different data. In the case of the black squadron pilots, the upgrades are still different, so a squad builder can distinguish the two quick build cards. It may be convenient to define an internal short ID field or other identification method, but this would be squad-builder specific. You could store it in the "vendor" field which is ignored by the xws spec.